charts/karpenter/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config
```

Comment:
Tekton and Knative use a common mechanism for this from knative/pkg that's worth looking into:

https://github.com/knative/serving/blob/main/pkg/apis/config/features.go

Reply:
I'm using the knative informer to monitor the ConfigMap and update any listeners. There is some parsing functionality in that package as well, but I preferred the warnings that I produce instead as I think they're more informative.